mvc/system/classes/rout.php

```diff
@@ -15,18 +15,17 @@ public function __construct()
     require_once "../application/controllers/" . $this->controller . ".php";
     $this->controller = new $this->controller;
 
-    if (!empty($url[1]) && method_exists($this->controller , $url[1])) {
+    if (!empty($url[1]) && method_exists($this->controller, $url[1])) {
       $this->method = $url[1];
       unset($url[1]);
     }
 
-    if(!empty($url)){
-      $this->params=$url;
+    if (!empty($url)) {
+      $this->params = $url;
     }
-
-    call_user_func_array([$this->controller , $this->method],$this->params);
+    call_user_func_array([$this->controller, $this->method], array($this->params));
   }
-
+  
   public function url()
   {
     if (isset($_SERVER["REQUEST_URI"])) {
```

Comment:
In line number 33 we can use `trim` function instead of ` rtrim ` function and include the "/" as the removable character. Cause in line 35 when using substring if an extra "/" is provided it will be considered thus resulting in an error, so we can just explode the URL rather than substring.